### DISCUSSION TITLE
Update to .NET 7

### DISCUSSION THREAD

**Discussion by Yushu2606** [ppy/osu#21742] [Ideas] [answered]
.NET 7 is released now

**Comment by ghost** [comment]
but why?

**Reply by Yushu2606** [reply]
https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-7

**Comment by smoogipoo** [comment, marked as answer]
It will happen when it happens. It's not a high priority right now.

**Comment by Minorsonek** [comment]
.NET 7 is not LTS, so in project of this scale it could be viable to just wait for .NET 8 which will be LTS
